src/nspawn/nspawn-mount.h

```diff
@@ -45,6 +45,7 @@ typedef struct CustomMount {
         char *type_argument; /* only for CUSTOM_MOUNT_ARBITRARY */
         bool graceful;
         bool in_userns;
+        uid_t dest_owner;
```

Comment:
this really should be initialized to UID_INVALID if not used, i.e. all other cases the structure is allocated.

Also, can you not abbreviate "dest" here, please. we kinda avoid that unless the abbreviation is super duper widely established.